docs/design/offline.md

```diff
@@ -123,13 +123,13 @@ It's okay if the app selects multiple streams of a type.  For text, this may be
 necessary, for example, to store subtitles for all languages.  For audio it may
 be necessary as well, for similar reasons.  For video it makes little sense, but
 we won't restrict it.  Apps can make whatever decision they want about tracks to
-store, but we should document best practice.
+store, but we should document best practices.
 
-We could also issue a warning if multiple tracks of same type, kind, and
-language are selected, since offline adaptation is currently an uncommon
+We could also issue a warning if multiple tracks of the same type, kind, and
+language is selected since offline adaptation is currently an uncommon
```

Comment:
No, "are" is right. "We could also issue a warning if multiple tracks ... are selected". It's plural.